clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerHeartbeatRequestManagerTest.java

```diff
@@ -380,6 +381,32 @@ public void testNetworkTimeout() {
         assertEquals(1, result.unsentRequests.size());
     }
 
+    @Test
+    public void testDisconnect() {
+        // The initial heartbeatInterval is set to 0
```

Comment:
I don't see much value in this comment (it's in the func name next line), should we remove it?

Reply:
Make sense